darts/utils/likelihood_models.py

```diff
@@ -171,6 +171,13 @@ def num_parameters(self) -> int:
         """
         pass
 
+    @abstractmethod
+    def __eq__(self, other) -> bool:
+        """
+        Define (in)equality between two likelihood objects
+        """
+        pass
+
```

Comment:
Attributes inheriting from torch.nn.Module need to be excluded because for example `nn.Softmax() == nn.Softmax()` returns `False`. Let me know what you think.